### PR TITLE
More trial and error, but now in Javascript

### PR DESCRIPTION
**Pull request by MerlinRose2000**

### CODE REVIEW

File: Develop/public/js/mainPage.js
Comment (Maximilian93B) [inline Develop/public/js/mainPage.js:1]: Hey Emilia , you almost got it but you are calling a few things incorrectly. So here is a revised version. For example. When you are using document.getElementByID  you dont need a function callback which is what you have, you are just trying to grab the element you are workking with/

// Wait for the document to fully load before executing the script
document.addEventListener("DOMContentLoaded", function () {
    // Select all elements that have the class 'state-border'
    var interactiveStates = document.querySelectorAll('.state-border');

    // Loop through each 'state-border' element
    interactiveStates.forEach(function (stateBorder) {
        // Add an event listener for mouseover actions on each border
        stateBorder.addEventListener('mouseover', function () {
            // Change the border's color to red when the mouse hovers over
            stateBorder.style.stroke = '#FF0000';
        });

        // Add an event listener for mouseout actions
        stateBorder.addEventListener('mouseout', function () {
            // Restore the border's color to black when the mouse leaves
            stateBorder.style.stroke = '#000';
        });
    });
})

**Review by Maximilian93B — CHANGES_REQUESTED** [review]
hey just check the comments

**Comment by MerlinRose2000** [comment]
trying this again

**Review by Maximilian93B — COMMENTED** [review]
looks good , keep working on it

**Pull request closed without merging**